src/include/common/statement.h

```diff
@@ -81,6 +75,10 @@ class Statement : public Printable {
 
   const std::shared_ptr<planner::AbstractPlan>& GetPlanTree() const;
 
+  std::unique_ptr<parser::SQLStatementList>const& GetStmtParseTreeList() {return sql_stmt_list_;}
```

Comment:
This method will be called when the statement lends its parse tree to the optimizer to generate a query plan. I think it may be better to keep this method now since the optimizer takes in a unique_ptr.